contracts/lib/GenesisLib.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+/**
+ * NOT_STARTED - child chain is not live, genesis validators can be added
+ * COMPLETED - child chain is live
+ */
+enum GenesisStatus {
+    NOT_STARTED,
+    COMPLETED
+}
+
+struct GenesisAccount {
+    address addr;
+    uint256 nonStakedTokens;
```

Comment:
Rename to `preminedTokens`?

Reply:
Fix: https://github.com/Ethernal-Tech/blade-contracts/pull/9/commits/f2942acb1332ddeb385a3107648dafeba77a8deb